ecl/resource_ecl_network_static_route_v2.go

```diff
@@ -162,20 +174,19 @@ func resourceNetworkStaticRouteV2Read(d *schema.ResourceData, meta interface{})
 	config := meta.(*Config)
 	networkClient, err := config.networkV2Client(GetRegion(d, config))
 	if err != nil {
-		return fmt.Errorf("Error creating ECL network client: %s", err)
+		return fmt.Errorf("error creating ECL network client: %w", err)
 	}
 
 	i, err := static_routes.Get(networkClient, d.Id()).Extract()
 	if err != nil {
 		return CheckDeleted(d, err, "static_route")
```

Comment:
他のエラーメッセージと揃えていただけますか？
```suggestion
		return CheckDeleted(d, err, "error getting ECL Static route")
```